src/main/java/driver/DriverFactory.java

```diff
@@ -1,13 +1,55 @@
 package driver;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 public class DriverFactory {
 
     public WebDriver driver;
+    public WebDriverWait wait;
 
     public DriverFactory(WebDriver driver) {
-
         this.driver = driver;
+        wait = new WebDriverWait(driver, 20);
+    }
+
+
+
+
+    @FindBy(id = "username")
+    private WebElement usernameField;
+
+    @FindBy(id = "password")
+    private  WebElement passwordField;
+
+    @FindBy (id = "log-in-button")
+    private  WebElement signInButton;
+    public void signInAsSuperAdmin() throws InterruptedException {
+        driver.get("https://zenistock.zenitech.local/login");
+        usernameField. sendKeys("super_admin");
+        passwordField.sendKeys("qwer3#qazxsw09*");
+        signInButton.click();
+        wait.until(ExpectedConditions.urlToBe("https://zenistock.zenitech.local/dashboard"));
```

Comment:
is there any way we could put all the credentials and links into a separate class and just call on them?